data/deep missions.txt

```diff
@@ -866,32 +881,33 @@ mission "Deep: Singularity 0A"
 			`	"If that is all then this meeting is over. You will be escorted back outside momentarily. Enjoy the rest of your day, Captain."`
 			
 			label next
-			`	The woman gets up from her chair and stops in the doorway. She turns her head slightly and says in a low voice, "I'll be watching," then leaves. Moments later, the same two men that brought you here step into the room and ask that you follow them. They lead you outside of the building and lock the door behind you as leave the building and step out into the spaceport.`
+			`	The woman gets up from her chair and stops in the doorway. She turns her head slightly and says in a low voice, "I'll be watching," then leaves. Moments later, the same two men that brought you here step into the room and ask you to follow them. They lead you to an entryway and lock the door as you step out into the spaceport.`
 				decline
 
 
 
 mission "Deep: Singularity 0B"
 	landing
 	name `Mysterious Message`
-	description `Travel to <destination>.`
+	description `Travel to <destination> to meet the author of an encrypted message.`
 	source
-		government Republic "Free Worlds" Syndicate Neutral
+		government Republic "Free Worlds" Syndicate Neutral Independent
+		near Sol 1 100
 	destination
 		attributes deep
 		attributes urban
 	to offer
+		random < 40
```

Comment:
She's going to know when you've helped the Deep almost immediately, so instead of a random offer, I think a delay (3 days to a week) between when Mystery Cubes 4 is completed and when this mission is offered would be a better way of going about things.